grpc.go

```diff
@@ -579,6 +608,13 @@ func (a *agentGRPC) CreateContainer(ctx context.Context, req *pb.CreateContainer
 		return emptyResp, err
 	}
 
+	// Change cwd because libcontainer sets the bundle path to cwd
+	oldcwd, err := pb.ChangeToBundlePath(ociSpec)
```

Comment:
These are only useful when there are hooks in the rootfs. So please make it optional by:
1. scan for rootfs hooks when agent gets up
2. change cwd and save spec in container bundle only when necessary